src/functions/template-tags/general.php

```diff
@@ -1113,8 +1114,8 @@ function tribe_events_event_schedule_details( $event = null, $before = '', $afte
 				$inner .= $end_date;
 			} else {
 				$inner .= tribe_get_start_date( $event, false, $format ) . ( $time ? $datetime_separator . tribe_get_start_date( $event, false, $time_format ) : '' );
-				$inner .= '</span>' . $time_range_separator;
-				$inner .= '<span class="tribe-event-date-end">';
+				$inner .= ($html ? '</span>' : '')  . $time_range_separator;
```

Comment:
Let's fix the spacing here

```suggestion
				$inner .= ( $html ? '</span>' : '' )  . $time_range_separator;
```